config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml

```diff
@@ -0,0 +1,57 @@
+presets:
+- labels:
+    preset-pull-kubernetes-e2e: "true"
```

Comment:
the presets are global, you can reference them from other files so you can drop them here.

Reply:
thanks removed